ExampleMod/Content/Prefixes/ExamplePrefix.cs

```diff
@@ -0,0 +1,67 @@
+using ExampleMod.Items;
+using Terraria;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Prefixes
+{
+    public class ExamplePrefix : ModPrefix
+    {
+        private readonly byte _power;
+
+        // See documentation for vanilla weights and more information.
+        // Note: a weight of 0f might still be rolled. See CanRoll to exclude prefixes.
+        // Note: if you use PrefixCategory.Custom, actually use ModItem.ChoosePrefix instead.
+        public override float RollChance(Item item)
+            => 5f;
```

Comment:
maybe use a ternary and have a lower roll chance for the higher power variant?
@JavidPack thoughts? Is this overcomplicating what should be a simple example?

Reply:
We can keep it simple, but a comment showing a switch would probably be good, since a real mod would probably make 4 copies